packages/styles/src/components/list-group.scss

```
@@ -38,125 +24,87 @@
   }
 
   .list-group-item-right-svg {
+    padding-block: tokens.get('list-icon-container-padding-block');
+    padding-inline: tokens.get('list-icon-container-padding-inline');
+
     @include transition(
       transform animation.$transition-time-area-small animation.$transition-easing-default
     );
   }
-
-  &-minimal,
-  &-transparent {
-    &:hover,
-    &:focus {
-      background-color: transparent;
-    }
-  }
-
-  &-inverted {
-    background-color: transparent;
-
-    &:hover,
-    &:focus {
-      background-color: list-group.$list-group-hover-bg-inverted;
-    }
-  }
 }
 
 .list-group-item {
```

If possible, omit this class.
```suggestion
> li {
```